lib/aliquot/validator.rb

```diff
@@ -64,6 +65,8 @@ def self.to_bool(lbd)
       predicate(:year?) { |x| x.between?(2000, 3000) }
 
       predicate(:base64_asn1?) { |x| OpenSSL::ASN1.decode(Base64.strict_decode64(x)) rescue false }
+
+      predicate(:json_object?) { |x| hash?(x) }
```

Comment:
How does this ensure that `x` is a JSON object?

Reply:
It doesn't, directly.

When parsing (`JSON.parse(some_json)`), JSON objects get turned into Ruby hashes.
Previously the error was something like `Must be a Hash`, which doesn't make sense in JSON terms, which is what the user supplies.
Now, the error is `must be a JSON object`.